config/betterquesting/DefaultQuests.json

```diff
@@ -352039,9 +351712,411 @@
+          "y:3": 168,
+          "id:3": 1922,
+          "sizeY:3": 24
+        },
+        "36:10": {
+          "sizeX:3": 24,
+          "x:3": -252,
+          "y:3": 168,
+          "id:3": 1923,
+          "sizeY:3": 24
+        },
+        "37:10": {
+          "sizeX:3": 24,
+          "x:3": -300,
+          "y:3": 168,
+          "id:3": 1924,
+          "sizeY:3": 24
+        },
+        "38:10": {
+          "sizeX:3": 24,
+          "x:3": -60,
+          "y:3": 264,
+          "id:3": 2148,
+          "sizeY:3": 24
+        },
+        "39:10": {
+          "sizeX:3": 24,
+          "x:3": -108,
+          "y:3": 264,
+          "id:3": 2149,
+          "sizeY:3": 24
+        },
+        "40:10": {
+          "sizeX:3": 24,
+          "x:3": -108,
+          "y:3": 312,
+          "id:3": 2150,
+          "sizeY:3": 24
+        },
+        "41:10": {
+          "sizeX:3": 24,
+          "x:3": -60,
+          "y:3": 456,
+          "id:3": 2586,
+          "sizeY:3": 24
+        },
+        "42:10": {
+          "sizeX:3": 24,
+          "x:3": -108,
+          "y:3": 456,
+          "id:3": 2591,
+          "sizeY:3": 24
+        },
+        "43:10": {
+          "sizeX:3": 24,
+          "x:3": -156,
+          "y:3": 456,
+          "id:3": 2700,
+          "sizeY:3": 24
+        },
+        "44:10": {
+          "sizeX:3": 24,
+          "x:3": -204,
+          "y:3": 312,
+          "id:3": 2731,
+          "sizeY:3": 24
+        },
+        "45:10": {
+          "sizeX:3": 24,
+          "x:3": -156,
+          "y:3": 216,
+          "id:3": 2732,
+          "sizeY:3": 24
+        },
+        "46:10": {
+          "sizeX:3": 24,
+          "x:3": -156,
+          "y:3": 312,
+          "id:3": 2733,
+          "sizeY:3": 24
+        },
+        "47:10": {
+          "sizeX:3": 24,
+          "x:3": -156,
+          "y:3": 360,
+          "id:3": 2734,
+          "sizeY:3": 24
+        },
+        "48:10": {
+          "sizeX:3": 24,
+          "x:3": -156,
+          "y:3": 408,
+          "id:3": 2735,
+          "sizeY:3": 24
+        },
+        "49:10": {
+          "sizeX:3": 24,
+          "x:3": -156,
+          "y:3": 264,
+          "id:3": 2894,
+          "sizeY:3": 24
+        },
+        "50:10": {
+          "sizeX:3": 24,
+          "x:3": 36,
+          "y:3": 120,
+          "id:3": 2971,
+          "sizeY:3": 24
+        },
+        "51:10": {
+          "sizeX:3": 24,
+          "x:3": -12,
+          "y:3": 120,
+          "id:3": 2972,
+          "sizeY:3": 24
+        },
+        "52:10": {
+          "sizeX:3": 24,
+          "x:3": -60,
+          "y:3": 120,
+          "id:3": 2973,
+          "sizeY:3": 24
+        },
+        "53:10": {
+          "sizeX:3": 24,
+          "x:3": -156,
+          "y:3": 504,
+          "id:3": 3235,
+          "sizeY:3": 24
+        }
+      },
+      "lineID:3": 33,
+      "properties:10": {
+        "betterquesting:10": {
+          "visibility:8": "NORMAL",
```

Comment:
Nitpick: the visibility is set to NORMAL, all other tabs have it set to UNLOCKED instead.

Reply:
uh yea, will fix next update